app/lib/world/world.coffee

```diff
@@ -260,6 +260,14 @@ module.exports = class World
         system.start @thangs
       catch err
         console.error "Error starting system!", system, err
+    # Silently clamp health after systems run.
+    hero = _.find(@thangs, (thang) -> thang.id == 'Hero Placeholder')
+    if hero?
+      if level.recommendedHealth?
+        hero.maxHealth = Math.max(hero.maxHealth, level.recommendedHealth)
+      if level.maximumHealth?
+        hero.maxHealth = Math.min(hero.maxHealth, level.maximumHealth)
+      hero.health = hero.maxHealth
```

Comment:
I just realized we only want to do this if we are in classroom mode, so when `level.type is 'course'`. In home mode, they still need to actually have that much health.